src/parser/components.rs

```diff
@@ -424,6 +448,60 @@ END:VEVENT
     );
 }
 
+#[test]
+fn test_multi_properties() {
+    let mut multi_properties: std::collections::BTreeMap<String, Vec<crate::properties::Property>> =
+        Default::default();
+    multi_properties.insert(
+        "ATTENDEE".into(),
+        vec![
+            Property {
+                name: "ATTENDEE".into(),
+                val: "mailto:email@example.com".into(),
+                params: vec![
+                    Parameter {
+                        key: "EMAIL".into(),
+                        val: Some("\"email@example.com\"".into()),
+                    },
+                    Parameter {
+                        key: "CUTYPE".into(),
+                        val: Some("INDIVIDUAL".into()),
+                    },
+                ],
+            }
+            .into(),
+            Property {
+                name: "ATTENDEE".into(),
+                val: "mailto:dmail@example.com".into(),
+                params: vec![
+                    Parameter {
+                        key: "EMAIL".into(),
+                        val: Some("\"dmail@example.com\"".into()),
+                    },
+                    Parameter {
+                        key: "CUTYPE".into(),
+                        val: Some("INDIVIDUAL".into()),
+                    },
+                ],
+            }
+            .into(),
+        ],
+    );
+    assert_parser!(
+        inner_component,
+        r#"
+BEGIN:VEVENT
+ATTENDEE;CUTYPE=INDIVIDUAL;EMAIL="email@example.com":mailto:email@example.com
+ATTENDEE;CUTYPE=INDIVIDUAL;EMAIL="dmail@example.com":mailto:dmail@example.com
+END:VEVENT
+"#,
```

Comment:
for simplicity you could also just test parsing and reserializing this same event

Reply:
Ok, interesting Idea. I will try this.

Reply:
nvm, don't have to, I tried, it's a bit of a waste of time ;)